forge-legacy/build.gradle.kts

```diff
@@ -28,7 +28,7 @@ val shade: Configuration by configurations.creating {
 dependencies {
     minecraft("com.mojang:minecraft:1.12.2")
     mappings("de.oceanlabs.mcp:mcp_stable:39-1.12")
-    forge("net.minecraftforge:forge:1.12.2-14.23.0.2486")
+    forge("net.minecraftforge:forge:1.12.2-14.23.5.2860")
```

Comment:
This may not work with Essential loom. If the build continues to fail, revert this.

Reply:
It should work fine I will test build later